crates/solidity/testing/sourcify/src/sourcify.rs

```diff
@@ -229,7 +262,9 @@ impl ContractArchive {
     }
 
     pub fn clean(self) {
```

Comment:
I wonder why are we cleaning in the first place? shouldn't we keep them by default?

- For CI runs, that is not needed, since they are ephemeral.
- For local runs, I expect us to be running a few shards in any case, and we can `cargo clean -p solidity_testing_sourcify` on demand in the rare case we bravely run all of them?